bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c

```diff
@@ -541,7 +575,7 @@ static int map_info_check(struct bpf_map_info *outter_info, struct bpf_map_info
 
     if (outter_info->max_entries < 2 || outter_info->max_entries > MAX_OUTTER_MAP_ENTRIES) {
         LOG_ERR("outter map max_entries must be in[2,%d]\n", MAX_OUTTER_MAP_ENTRIES);
-        return -EINVAL;
+        // return -EINVAL;
```

Comment:
Should we update the if check now? IIUC The max_entries can exceed MAX_OUTTER_MAP_ENTRIES after scale out

Reply:
So should this be equal to MAX_OUTTER_MAP_ENTRIES

Reply:
```c
struct {
    __uint(type, BPF_MAP_TYPE_ARRAY_OF_MAPS);
    __uint(key_size, sizeof(__u32));
    __uint(value_size, sizeof(__u32));
    __uint(max_entries, MAP_SIZE_OF_OUTTER_MAP);
    __uint(map_flags, 0);
} outer_map SEC(".maps");
```

The maximum number of records has been specified when outter_map is defined. Here, only verification is performed.